yarn-project/types/src/abi/contract_artifact.ts

```diff
@@ -125,6 +135,19 @@ function generateFunctionArtifact(fn: NoirCompiledContractFunction): FunctionArt
   };
 }
 
+function getFunctionType(fn: NoirCompiledContractFunction): FunctionType {
+  if (fn.custom_attributes.includes(AZTEC_PRIVATE_ATTRIBUTE)) {
+    return FunctionType.SECRET;
+  } else if (fn.custom_attributes.includes(AZTEC_PUBLIC_ATTRIBUTE)) {
```

Comment:
```suggestion
  } else if (fn.custom_attributes.includes(AZTEC_PUBLIC_VM_ATTRIBUTE)) {
```

Here's the issue. Previously the `#[aztec(public-vm)]` attribute would set the `is_open` flag but now we're looking for the `#[aztec(public)]` attribute when selecting functions to run on the AVM.